cmd/blocking-issue-creator/main.go

```diff
@@ -190,6 +191,7 @@ func main() {
 			}
 
 			logger.Infof("Updated issue %d", updateIssue.UpdateIssue.Issue.Number)
+			time.Sleep(time.Second * 1)
```

Comment:
Can we just use the prow gh client and enable the throttler on it?

Reply:
Yea I will take a look and do the changes. I hold the PR

Reply:
Just did the math - we need to make ~200 calls once an hour, so a sleep is trivially correct (although the sleeps in this PR are not - we need to wait after the search). If it's easy to do the throttler, I'd prefer that